installer/Migration/V4_9_0/Migration.php

```diff
@@ -0,0 +1,154 @@
+        $this->getSchemaHelper()->addColumn('ohrm_theme', 'social_media_icons', Types::TEXT, [
+            'Notnull' => true,
+            'Default' => 'inline',
+        ]);
+        $this->getSchemaHelper()->addColumn('ohrm_theme', 'login_banner', Types::BLOB);
+
+        $this->createQueryBuilder()
+            ->insert('ohrm_i18n_group')
+            ->values(
+                [
+                    'name' => ':name',
+                    'title' => ':title',
+                ]
+            )
+            ->setParameter('name', 'branding')
+            ->setParameter('title', 'Corporate Branding')
+            ->executeQuery();
+
+        $this->createQueryBuilder()
+            ->delete('ohrm_marketplace_addon')
+            ->andWhere('ohrm_marketplace_addon.plugin_name = :pluginName')
+            ->setParameter('pluginName', 'orangehrmCorporateBrandingPlugin')
+            ->executeQuery();
+
+        $this->updateConfig('4.9', 'instance.version');
+        $this->updateConfig('81', 'instance.increment_number');
+
+        if (!$this->getSchemaHelper()->tableExists(['ohrm_registration_event_queue'])) {
+            $this->getSchemaHelper()->createTable('ohrm_registration_event_queue')
+                ->addColumn('id', Types::INTEGER, ['Autoincrement' => true])
+                ->addColumn('event_type', Types::INTEGER, ['Notnull' => true])
+                ->addColumn('published', Types::SMALLINT, ['Unsigned'=> true,'NotNull' => true,'Default' => 0])
+                ->addColumn('event_time', Types::DATETIME_MUTABLE, ['Default'=> null,'Notnull' => true])
+                ->addColumn('publish_time', Types::DATETIME_MUTABLE, ['Default'=> null,'Notnull' => true])
+                ->setPrimaryKey(['id'])
+                ->create();
+        }
+    }
+
+    /**
+     * @param string $menuItem
+     * @param int $screenId
+     * @param int $parentId
+     * @param int $level
+     * @param int $order_hint
+     * @param string $urlExtras
+     * @param int $status
+     * @return void
+     */
+    private function insertMenuItems(
+        string $menuItem,
+        string $screenName,
+        int    $parentId,
+        int    $level,
+        int    $order_hint,
+        string $urlExtras,
+        int    $status
+    ): void {
+        $screenId = $this->getConnection()->createQueryBuilder()
+            ->select('screen.id')
+            ->from('ohrm_screen', 'screen')
+            ->where('screen.name = :screenName')
+            ->setParameter('screenName', $screenName)
+            ->executeQuery()
+            ->fetchOne();
+
+        $this->createQueryBuilder()
+            ->insert('ohrm_menu_item')
+            ->values(
+                [
+                    'menu_title' => ':menuTitle',
+                    'screen_id' => ':screenId',
+                    'parent_id' => ':ParentId',
```

Comment:
```suggestion
                    'parent_id' => ':parentId',
```
minor issue,
`->setParameter('parentId', $parentId)`

Reply:
changed